### PR TITLE
OpenShift deployment (v1; more to come)

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

**Comment by vorburger** [comment]
@edewit how do you like this? :smiley: OK for if I merge this as-is?

File: README.md [hunk 1]
```diff
@@ -69,9 +69,23 @@ You can obviously mix the order and repeat titles, comments, chats, narrations,
...
+
     docker build -t minecraft-storeys-maker .
 
-    docker run -p 25565:25565 -p 8080:8080 -p 9090:9090 minecraft-storeys-maker
+    docker run --rm -p 25565:25565 -p 8080:8080 -p 7070:7070 minecraft-storeys-maker
+
+In OpenShift, just ignore the "error: build error: No source files were specified" from the first command:
+
+    oc new-app "https://github.com/vorburger/minecraft-storeys-maker.git#container"
+
```
Comment (edewit) [inline README.md]: you would need to change this once this is merged right?

**Comment by edewit** [comment]
looks really nice

File: README.md (hunk 1 above)
Comment (vorburger) [reply README.md]: oups yes, I'll fix that up right away

Comment (vorburger) [reply README.md]: well spotted - thanks - fixed up now - OK for if I merge this as-is?

**Comment by edewit** [comment]
:+1:

**Pull request merged**